dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkflowDefinitionServiceImpl.java

```diff
@@ -1438,7 +1437,7 @@ private TaskDefinitionLog buildNormalSqlTaskDefinition(String taskName, DataSour
         sqlParameters.setLocalParams(Collections.emptyList());
         taskDefinition.setTaskParams(JSONUtils.toJsonString(sqlParameters));
         taskDefinition.setCode(CodeGenerateUtils.genCode());
-        taskDefinition.setTaskType(SqlTaskChannelFactory.NAME);
+        taskDefinition.setTaskType("SQL");
```

Comment:
We need to use enum or constant, otherwise once the type has been changed, it's hard to know we need to make change here.

Reply:
This function is useless, I will remove this piece in other PR.

Reply:
Please link issue to track this.

Reply:
Ok